assignment3/src/problem1_UsingES6.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>ES Demo</title>
+        <script>
+            function printNumbers() {
+                const WAIT_TIME = 1000;
+                for (let iterator = 1; iterator < 10; iterator++) {
+                    function timeout(iterator) {
+                        setTimeout(() => document.write(iterator + '<br>'), iterator * WAIT_TIME);
+                    }
+                    timeout(iterator);
```

Comment:
In the case where we define and function and use it, we can go with an anonymous function.